src/ert/gui/ertwidgets/ensembleselector.py

```diff
@@ -65,13 +71,21 @@ def populate(self) -> None:
 
         self.clear()
 
-        if self._ensemble_list():
-            self.setEnabled(True)
+        try:
+            if self._ensemble_list():
+                self.setEnabled(True)
 
-        for ensemble in self._ensemble_list():
-            self.addItem(
-                f"{ensemble.experiment.name} : {ensemble.name}", userData=ensemble
-            )
+            for ensemble in self._ensemble_list():
+                self.addItem(
+                    f"{ensemble.experiment.name} : {ensemble.name}", userData=ensemble
+                )
+        except OSError as err:
+            logger.error(str(err))
+            Suggestor(
+                errors=[ErrorInfo(str(err))],
+                widget_info='<p style="font-size: 28px;">Error writing to storage</p>',
+            ).show()
```

Comment:
Would it be a bad idea to wrap this as a utility function?
You'd only need two inputs, the error itself, and the "Error writing to storage" string. All the styling and sizing could be prepended in the function.

Reply:
Yes, it absolutely makes sense, but maybe in a later PR? Not obvious if the supporting functionality should be in `ensembleselector.py` or in the suggestor itself?